src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -3797,6 +3809,7 @@ Activate by holding the key for the character you want to add an accent to, then
   </data>
   <data name="RegistryPreview.ModuleDescription" xml:space="preserve">
     <value>A quick little utility to visualize and edit complex Windows Registry files.</value>
+	<comment>{Locked="Windows"}</comment>
```

Comment:
please make sure you use spaces instead of tabs -- re-save this file in visual studio's resw editor